src/open_inwoner/accounts/views/cases.py

```diff
@@ -347,8 +350,14 @@ def get_context_data(self, **kwargs):
                 "end_date_legal": getattr(
                     self.case, "uiterlijke_einddatum_afdoening", None
                 ),
-                "description": self.case.omschrijving,
-                "type_description": self.case.zaaktype.omschrijving,
+                "description": (
+                    self.case.omschrijving
+                    if self.case.omschrijving
+                    else self.case.zaaktype.omschrijving
+                ),
+                "type_description": (
+                    self.case.zaaktype.omschrijving if self.case.omschrijving else ""
```

Comment:
I note the condition looks at a different attribute then the left side but suppose it is intentional and related to the regular `description` just above.